expression/builtin_vectorized_test.go

```diff
@@ -136,22 +136,31 @@ func (s *testVectorizeSuite2) TestMockVecPlusIntParallel(c *C) {
 	wg.Wait()
 }
 
+const (
+	initTestBufCap = 65536
+	numTestGet     = 4096
+)
+
 func BenchmarkColumnBufferAllocate(b *testing.B) {
-	allocator := newLocalSliceBuffer(1)
+	allocator := newLocalSliceBuffer(initTestBufCap)
 	b.ResetTimer()
 	for i := 0; i < b.N; i++ {
-		buf, _ := allocator.get(types.ETInt, 1024)
-		allocator.put(buf)
+		for j := 0; j < numTestGet; j++ {
+			col, _ := allocator.get(types.ETInt, chunk.InitialCapacity)
+			allocator.put(col)
```

Comment:
IMO, benchmark for the allocating should not release buffer after allocate immediately.  In this case, it's only allocated a chunk once and always reuse it.

Reply:
That is the original logic. I think it's also persuasive to benchmark `get` and `put` without alloc

Reply:
👌